sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -349,6 +353,13 @@ void CNPC_Combine::ClearFollowTarget()
 //-----------------------------------------------------------------------------
 void CNPC_Combine::Use(CBaseEntity *pActivator, CBaseEntity *pCaller, USE_TYPE useType, float value)
 {
+#ifdef EZ
+	if ( m_bDisablePlayerUse )
+	{
+		return;
+	}
+#endif
+
 	m_OnPlayerUse.FireOutput(pActivator, pCaller);
```

Comment:
Could this be moved below `OnPlayerUse` so that the output could be used in connection with this feature? (e.g. map logic has a soldier do/say something to indicate why they can't be added/removed from squad)

```suggestion
	m_OnPlayerUse.FireOutput(pActivator, pCaller);

#ifdef EZ
	if ( m_bDisablePlayerUse )
	{
		return;
	}
#endif
```